### DISCUSSION TITLE
Option to select Interface language

### DISCUSSION THREAD

**Discussion by FelicityIris** [spacecowboy/Feeder#133] [Ideas]
My device does not support the language I desire to use my applications in, hence I have to default to English which the application uses and then I need to rely on the application itself to provide me the option of choosing a language. I wish there were an option to select the UI language inside the Feeder application itself to mitigate this issue.

**Comment by spacecowboy** [comment]
Feeder will not support languages your device does not so this won't help you.

You can set language used in Feeder by following instructions here (if your version of Android isn't too old):
https://developer.android.com/guide/topics/resources/app-languages

**Reply by FelicityIris** [reply]
My OS is android 12 and does not include the Esperanto language in the global language settings unlike Android 13. I had already started translating the app to Esperanto on weblate so this is quite disheartening to read because I had convinced multiple apps to add such an option in the past. (Replying inside thread this time.)

Such an option can indeed help me as it has in other apps which support such a setting.

**Reply by spacecowboy** [reply]
https://developer.android.com/guide/topics/resources/app-languages

only supported in Android 13.

**Reply by FelicityIris** [reply]
It would be helpful if support is added for users not on Android 13 because I can not afford an Android 13 device at the moment and this essentially handicaps me.

**Comment by Coool** [comment]
I'm almost done 50% translation, but all the strings are still in English. Is the translation included when it's 100% translated? Or it's old Android issue? You **must** [^requirement_language] have Android 13 when was introduced per App language setting.

[^requirement_language]: You need to mention in README.md

**Reply by spacecowboy** [reply]
Translations show up as pull requests: https://github.com/spacecowboy/Feeder/pull/756

They are then included in the app once they reach above a certain threshold, I forget the exact number but I do exclude translations which are too incomplete

**Reply by Coool** [reply]
@spacecowboy, can you change to Latvian language on your phone? And which Android version you are using?

**Reply by spacecowboy** [reply]
Doesn't seem to be in the app. I only see Lithuanian

**Reply by Coool** [reply]
Installed 2.11.5[^2115] GitHub F-Droid release and I have Latvian language now. Now I can continue my translation journey. 😉

[^2115]: [2.11.5](https://github.com/spacecowboy/Feeder/releases/tag/2.11.5)
